index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Tangerine|Inconsolata|Droid+Sans|Poppins">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700:900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./index.css">
+    <title>Porfolio</title>
+</head>
+<body>
+    <div class="wrapper">
+        <header id="header"> 
+            <div class="logo blue-text"><a href="#">Joeveloper</a></div>
+            <nav class="nav-bar">
+                <ul id="navlink-ul">
+                    <li><a class="nav-links black-text" href="">Portfolio</a></li>
+                    <li><a class="nav-links black-text" href="">About</a></li>
+                    <li><a class="nav-links black-text" href="">Contact</a></li>
+                    <li><a class="nav-links black-text" href=""><i class="fas fa-envelop"></i></a></li>
+                </ul>
+                <div class="burger-parents">
+                    <div class="hamburger"></div>
+                    <div class="hamburger"></div>
+                    <div class="hamburger"></div>
+                </div>
+                
+        </nav>
+    </header>
+
+    <section id="portfolio-main">
+        <div id="div">
+            <div class="main-intro">
+                <div id="maintext-body">
+                    <h1 class="intro-heading .black-text">I'm Joeveloper</h1>
+                    <h1 class="intro-heading">Glad to see you!</h1>
+                    <p id="intro-para" class="black-text">I'm a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hesitate to contact me.</p>
+                </div>
+            </div>
+            <div id="social-media">
+                <p id="social-para" class="social-blue">LET'S CONNECT</p>
```

Comment:
- [ ] **As suggested by the previous reviewer,** It would be better to use a link ```<a>``` or ```<button>``` tag, and apart from being semantically correct, the ```a``` tag will let you connect to other sections of the website, using ```href``` attribute, 😎😎

example:
```
<a href="#" id="social-para" class="social-blue">LET'S CONNECT</a>
```

You may follow this [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) for more info on HTML Links 🙏